LayoutTests/imported/w3c/web-platform-tests/css/css-tables/out-of-order-elements-collapsed-border-ref.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<title>CSS Test: Table border is correct even with out of order tbody, thead, tfoot</title>
+<link rel="author" title="Karl Dubost" href="mailto:karlcow@apple.com">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/tables.html#forming-a-table">
+<link rel="match" href="out-of-order-elements-collapsed-border-ref.html">
```

Comment:
These are not useful in a -ref.html file :) (same comment about -expected.html)

Reply:
nice catch! Thanks.

Reply:
changed!